src/modules/md/md.cpp

```diff
@@ -30,12 +31,12 @@ MDModule::MDModule() : Module("MD")
                                onlyWhenEnergyStable_);
 
     keywords_.setOrganisation("Options", "Output");
-    keywords_.add<IntegerKeyword>("EnergyFrequency", "Frequency at which to calculate total system energy (or 0 to inhibit)",
-                                  energyFrequency_, 0);
-    keywords_.add<IntegerKeyword>("OutputFrequency", "Frequency at which to output step information (or 0 to inhibit)",
-                                  outputFrequency_, 0);
-    keywords_.add<IntegerKeyword>("TrajectoryFrequency", "Write frequency for trajectory file (or 0 to inhibit)",
-                                  trajectoryFrequency_, 0);
+    keywords_.add<OptionalIntegerKeyword>("EnergyFrequency", "Frequency at which to calculate total system energy",
+                                          energyFrequency_, 0, std::nullopt, 10, "Off");
```

Comment:
For consistency, let's make all the value steps `5`:
```suggestion
                                          energyFrequency_, 0, std::nullopt, 5, "Off");
```